docs/pages/get-started/create-a-project.mdx

```diff
@@ -17,7 +17,13 @@ To create a new project, run the following command:
 
 <Terminal cmd={['$ npx create-expo-app@latest']} />
 
-> You can choose a different template by adding the `--template` flag.
+### Project templates
+
+The default project template created by `npx create-expo-app` includes [Expo Router](/router/introduction/), the fastest way to get started building a multi-screen app with Expo. However, you can choose a different template by adding the `--template` flag.
+
+To create an app with only an **App.js** file that you can later add your own navigation solution to, use the `blank` template:
+
+<Terminal cmd={['$ npx create-expo-app@latest --template blank']} />
```

Comment:
```suggestion
> The default project template includes [Expo Router](/router/introduction/) library installed to build multi-screen apps with Expo. You can choose another template by using the `--template` option. For example, using this option, you can choose a `blank` template to create a project with minimum required libraries installed.
```

Let's update this inside the callout instead of adding a new section. The point of having this callout was to not distract new users from the new onboarding flow they might follow on expo.dev along with this section of docs. Also, there are other templates, and in some cases, a user may want to use the blank template with TypeScript instead of the JS one. I also see this as an educative point where if we tell new users about --the template option, they run the command and see what other templates might fit their requirements. The users already familiar with this command can use whichever template they want.

(_I've task to document create-expo-app and can explore all about template in detail there and link that here soon._)